rustbpe/src/lib.rs

```diff
@@ -256,6 +256,12 @@ impl Tokenizer {
     }
 }
 
+impl Default for Tokenizer {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
```

Comment:
While I know that this is a standard Rust convention, I don't really see the added value of having this `Default` implementation in this case.

Reply:
I believe `Default` is necessary if this Rust code would be used with test helpers and various Rust API's. But it's not needed for Python usage.